shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -32,6 +32,18 @@ @interface FlutterViewController () <FlutterBinaryMessenger>
 @property(nonatomic, readwrite, getter=isDisplayingFlutterUI) BOOL displayingFlutterUI;
 @end
 
+#if __IPHONE_OS_VERSION_MAX_ALLOWED < 130000
+typedef enum UIAccessibilityContrast : NSInteger {
+  UIAccessibilityContrastUnspecified = 0,
+  UIAccessibilityContrastNormal = 1,
+  UIAccessibilityContrastHigh = 2
+} UIAccessibilityContrast;
+
+@interface UITraitCollection (AccessibilityContrastApi)
```

Comment:
Nit: I'd suggest naming the extension something like `MethodsFromNewerSDK`, rather than `AccessibilityContrastApi`, to make it very obvious that this is a declaration of something from the SDK rather than a normal extension (with an implementation).

If you do keep the current name, s/Api/API/ per ObjC naming style.

Reply:
done